src/components/pages/ImageClassification/ImageClassificationTable/ImageClassificationObservationTable.js

```diff
@@ -319,53 +315,6 @@ const ImageClassificationObservationTable = ({
       .sort(sortByLatest)
   }, [distillAnnotationData, images, numPointsPerQuadrat])
 
-  // Poll every 5 seconds after the first image is uploaded
-  const _pollImageStatuses = useEffect(() => {
-    let intervalId
-
-    const startPolling = async () => {
-      try {
-        const response = await databaseSwitchboardInstance.getAllImagesInCollectRecord(
-          projectId,
-          recordId,
-          EXCLUDE_PARAMS,
-        )
-
-        setImages(response.results)
-
-        const areAllImagesProcessed = response.results.every((file) =>
-          isImageProcessed(file.classification_status?.status),
-        )
-
-        if (areAllImagesProcessed) {
-          setPolling(false)
-        } else {
-          intervalId = setTimeout(startPolling, 5000)
-        }
-      } catch (error) {
-        handleHttpResponseError({
-          error,
-          callback: () => {
-            console.error('Error polling images:', error)
-          },
-          shouldShowServerNonResponseMessage: false,
-        })
-        intervalId = setTimeout(startPolling, 5000)
-      }
-    }
-
-    if (polling) {
-      startPolling()
-    }
-
-    return () => {
-      if (intervalId) {
-        clearTimeout(intervalId)
-      }
-    }
-    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
this effect was overreactive causing excessive polling, even with ignored deps (This is a lint rule I never ignore)